ORGANIZATION.md

```diff
@@ -0,0 +1,83 @@
+# Our organization
+
+At Fortytwo, we operate with a "flat organization" structure, which means we keep the number of management and overhead levels to a minimum. We believe that having flexible, autonomous, and self-service teams for customer delivery and product development is the best way to serve our customers and employees. This structure allows all of our employees to be part of a team and product development that is bigger than themselves, which results in better performance, better work-life balance, and higher-quality deliverables for our customers.
+
+This document will explain the idea and structure of the teams. Also the different roles.
+
+## Our common expectations 
+To ensure clear communication and alignment, here is a description of the expectations for each role within the company.
+
+### What we expect from all engineers
+
+All engineers are expected to know, and/or make sure to train for the following knowledge as a minimum:
+
+* Basic Git knowledge.
+* Basic Azure DevOps understanding and usage.
+* Basic Markdown syntax and understanding for documentation.
+* Basic SCRUM knowledge.
+* Basic knowledge with Powershell or C#
```

Comment:
Could we change this to something less specific? Like, `Basic knowledge of scripting and programming languages`?